testing/scenario_app/lib/src/channel_util.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:ui';
+
+import 'package:meta/meta.dart';
+
+/// Util method to replicate the behavior of a `MethodChannel` in the Flutter
+/// framework.
+void sendMethodCall({
```

Comment:
We use the json codec typically not the standard message codec?  maybe the name should reflect json or take in an optional argument of the codec?

Reply:
oh ya good call. Only our system channels are being weird with json codecs. Good point. Name changed.